iocage/lib/Config/Jail/BaseConfig.py

```diff
@@ -150,7 +150,7 @@ def _set_legacy(  # noqa: T484
             self.legacy = False
 
     def _get_id(self) -> str:
-        return str(self.data["id"])
+        return iocage.lib.helpers.to_string(self.data["id"])
```

Comment:
oops

```
./iocage/lib/Config/Jail/BaseConfig.py:153:10: T484 Returning Any from function declared to return "str"
```

Reply:
Just wrap `str()` around it - that should satisfy the type check.